subprojects/workers/src/main/java/org/gradle/workers/WorkerConfiguration.java

```diff
@@ -25,7 +25,28 @@
 import java.io.File;
 
 /**
- * Represents the configuration of a worker.
+ * Represents the configuration of a worker.  Typically used when submitting an item of work
+ * to the {@link WorkerExecutor}.
+ *
+ * <pre>
+ *      workerExecutor.submit(RunnableWorkImpl.class) { WorkerConfiguration conf ->
+ *          // Set the isolation mode for the worker
```

Comment:
⭕️ I personally don't like these kinds of comments that just state the obvious. They often just make the code harder to follow.